src/main/java/spoon/support/sniper/internal/ElementSourceFragment.java

```diff
@@ -7,6 +7,7 @@
  */
 package spoon.support.sniper.internal;
 
+
```

Comment:
You accidentally added this new line.

Reply:
Yes, I still have problems of added newlines while sniper printing. I removed it.